site/en/docs/android/trusted-web-activity/whats-new/index.md

```diff
@@ -0,0 +1,429 @@
+This will return a service instance for Google Play Billing and other vendors can implement support
+for the Digital Goods API and will have different identifiers.
+
+Finally, call `getDetails()` on the reference to the Google Play Billing object passing the SKU for
+the item as a parameter. The method will return a detail object containing both the price and the
+currency for the item that can be displayed to the user.
+
+#### Start the purchase flow
+
+The Payment Request API enables purchase flows on the web and is also used for the Google Play
+Billing integration. Check out this [How Payment Request API Works][16] to learn more if you are new to the Payment
+Request API.
+
+In order to use the API with Google Play Billing you will need to add a payment instrument which
+has a supported metod called `https://play.google.com/billing` and add the SKU as part of the data
+for the instrument:
+
+```javascript
+const supportedInstruments = [{
+  supportedMethods: "https://play.google.com/billing",
+  data: {
+    sku: sku
+  }
+}]
+```
+
+Then, build a `PaymentRequest` object as usual and use the API as usual
+
+```javascript
+const request = new PaymentRequest(supportedInstruments, details);
+```
+
+#### Acknowledge the purchase
+
+Once the transaction is complete, you will need to use the Digital Goods API to acknowledge the
+payment. The response object from the `PaymentRequest` will contain a token you will use to
+acknowledge the transaction:
+
+```javascript
+const response = await request.show();
+const token = response.details.token;
+const service =
+          await window.getDigitalGoodsService("https://play.google.com/billing");
+await service.acknowledge(token, 'onetime')
+```
+
+The Digital Goods API and the Payment Request API don't have knowledge on the user's identity. As a
+result, it's up to you to associate the purchase to the user in your backend and ensure they have
+access to the purchased items. When associating the purchase to a user, remember to save the
+purchase token, as you may need it to verify if the purchase has been cancelled or refunded, or if
+a subscription is still active. Check out the [Real Time Developer Notifications API][17] and the
+[Google Play Developer API][15] as they provide endpoints for handling those cases in your backend.
+
+#### Check for existing entitlements
+
+A user may have redeemed a promo code or may have an existing subscription to your product. In
+order to validate that the user has the appropriate entitlements, you can call the
+`listPurchases()` command on the digital goods service. This will return all purchases that your
+customer has made in your app. This would also be the place to acknowledge any unacknowledged
+purchases to ensure that the user correctly redeems their entitlements.
+
+```javascript
+const purchases = await itemService.listPurchases();
+for (p of purchases) {
+  if (!p.acknowledged) {
+    await itemService.acknowledge(p.purchaseToken, “onetime”);
```

Comment:
This has smart quotes. I'm actually surprised the linter isn't complaining. Can you remove them?

Reply:
Thanks for catching this. I'll remove the smart quotes - I'm guessing the linter is not complaining because it's inside a code block?

Reply:
Replaced the smart quotes with single quotes.